frontend/www/js/omegaup/components/login/Signup.test.ts

```diff
@@ -0,0 +1,41 @@
+import { shallowMount } from '@vue/test-utils';
+
+import login_Signup from './Signup.vue';
+
+describe('signup.vue', () => {
```

Comment:
You will need add the mock for `Date.now()`, because you need to check when a user is Under13 or Over13
```suggestion
describe('signup.vue', () => {
  // Setting an specific datetime to avoid flakiness in a leap-year
  const now = new Date('2022-01-01T00:00:00Z').getTime();
  let dateNowSpy: jest.SpyInstance<number, []> | null = null;

  beforeEach(() => {
    dateNowSpy = jest.spyOn(Date, 'now').mockImplementation(() => now);
  });

  afterEach(() => {
    if (dateNowSpy) {
      dateNowSpy.mockRestore();
    }
  });
```